tests/e2e/README.md

```
@@ -65,38 +92,56 @@ bash scripts/qa-smoke.sh live
 
 # Full trace on every test (huge, debugging only)
-PWDEBUG=1 pytest tests/e2e/test_smoke.py -v
+PWDEBUG=1 pytest tests/e2e/ -v
 ```
 
 ## Output
 
-Pass:
+Pass (23 tests across the four files):
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)